### PR TITLE
docs: update Hacktoberfest section to a dropdown

### LINKED ISSUE #798: Hacktoberfest info on README.md
### What would you like to share?

I was wondering if it is neccessary to keep the hacktoberfest notice on the readme, right now, as it is not hacktoberfest until next year.
image

Maybe, we could make it a dropdown, like this: 

<details close>
  <summary> Note for Hacktoberfest</summary>
  <br>

> A note for Hacktoberfest Participants:
>
> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
>
> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
  </details>

The code:

```
<details close>
  <summary> Note for Hacktoberfest</summary>
  <br>

> A note for Hacktoberfest Participants:
>
> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
>
> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
  </details>
```

### Additional information

I don't know is this is needed, but, this is an idea, for first time contributors.
Thank you.

### PR DESCRIPTION
**Pull request by arghyab0**
## Fixes Issue

Fixes  #798 

## Changes proposed

- Not neccessary to keep the Hacktoberfest notice on the README during this time of the year
- Shifted the notice to a dropdown

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/803"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,14 +5,17 @@
...
-## Note for Hacktoberfest
-
-> A note for Hacktoberfest Participants:
->
-> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
->
-> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
-> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
+<details close>
+  <summary><span style="font-size: 20px; font-weight: 600;">Note for Hacktoberfest</span></summary>
```
Comment (CodingSpecies) [inline README.md:9]: - @arghyab0 I think, in my opinion, we do not need the span styling.
- Also, please format the code on line 9, so there is a space between the `>` and `Note`also the word `Hacktoberfest` and `</span>`.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (arghyab0) [reply README.md]: Hey @CodingSpecies, thanks for the review. I have implemented the suggested changes:
- Removed the span styling
- Formatted the code to include spaces between the tags and the content
Cheers!

**Review by CodingSpecies — APPROVED** [review]
@arghyab0 Thank you for implementing the requested changes!
LGTM!

**Review by CodingSpecies — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -5,14 +5,17 @@
...
-## Note for Hacktoberfest
-
-> A note for Hacktoberfest Participants:
->
-> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
->
-> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
-> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
+<details close>
+  <summary> Note for Hacktoberfest </summary>
```
Comment (Panquesito7) [inline README.md:9]: I think we should add bold here or header.
```suggestion
  <summary><b> Note for Hacktoberfest </b></summary>
```

**Review by CodingSpecies — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (arghyab0) [reply README.md]: I agree, @Panquesito7. It makes the dropdown between two images a bit more noticeable. I have implemented the same.

File: README.md [hunk 3]
```diff
@@ -5,14 +5,17 @@
...
-## Note for Hacktoberfest
-
-> A note for Hacktoberfest Participants:
->
-> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
->
-> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
-> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
+<details close>
+  <summary><b> Note for Hacktoberfest </b></summary>
```
Comment (Panquesito7) [inline README.md:9]: Do you think the header would look fine? I think it'd be nice.
```suggestion
  <summary><h2> Note for Hacktoberfest <h2></summary>
```

Comment (arghyab0) [reply README.md:9]: Unfortunately, adding a header tag would push the content to the next line and the dropdown arrow would be above it. Here's the screenshot: 

image

However, we can achieve the same effect of a header tag by including some inline styling. Here's what that would look like:

image

Please let me know which one you would prefer and I'll make the changes accordingly.

Comment (Panquesito7) [reply README.md:9]: The second option looks good. 👍

Comment (Panquesito7) [reply README.md:9]: The second option looks good. 👍

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

File: README.md (hunk 3 above)
Comment (CodingSpecies) [reply README.md]: @arghyab0 @Panquesito7 now it has been committed, in the readme the styling to make it into a header has not been effective, as it still looks like a paragraph text.

Comment (arghyab0) [reply README.md]: @CodingSpecies I just checked and it seems that GitHub sanitises the HTML and removes inline styling amongst other things. 
We could do something like this: 

image

Another option would be to have a separate MD file and link it to the heading. Here's what that would look like:

image

**Pull request merged**